src/features/lightspeed/roleGeneration.ts

```diff
@@ -93,6 +94,20 @@ async function generateRole(
   }
 }
 
+async function getCollectionsFromWorkspace() {
+  const workspaceFolders = workspace.workspaceFolders;
+  let collectionsFound: string[] = [];
+  if (workspaceFolders) {
+    const workspaceDirectories = workspaceFolders.map((f) => f.uri.fsPath);
+    const collectionFinder = new CollectionFinder(workspaceDirectories[0]);
```

Comment:
I read ["Multi-root workspaces"](https://code.visualstudio.com/docs/editor/workspaces#_multiroot-workspaces) and wondered there might be more than workspaceDirectories.  I think 99.99% of users are not using it wit Ansible extension and do not think we need to support those cases, but we may want to add `console.debug()` when `workspaceDirectories.length > 1`

Reply:
Ah thanks! I forgot about this one. I just adjusted the code to accept a list of workspace directories.